<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostedListDialog</class>
 <widget class="QWidget" name="PostedListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="GroupTreeWidget" name="groupTreeWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QFrame" name="frame">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="submitPostButton">
            <property name="text">
             <string>Submit a new Post</string>
            </property>
            <property name="icon">
             <iconset resource="Posted_images.qrc">
              <normaloff>:/images/posted_24.png</normaloff>:/images/posted_24.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Todo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="showLabel">
            <property name="text">
             <string>Showing 1-100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="prevButton">
            <property name="text">
             <string>Prev</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="sortGroup">
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="hotSortButton">
              <property name="text">
               <string>Hot</string>
              </property>
              <property name="icon">
               <iconset resource="Posted_images.qrc">
                <normaloff>:/images/hot_24.png</normaloff>:/images/hot_24.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="newSortButton">
              <property name="text">
               <string>New</string>
              </property>
              <property name="icon">
               <iconset resource="Posted_images.qrc">
                <normaloff>:/images/new_24.png</normaloff>:/images/new_24.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="topSortButton">
              <property name="text">
               <string>Top</string>
              </property>
              <property name="icon">
               <iconset resource="../images.qrc">
                <normaloff>:/images/vote_up.png</normaloff>:/images/vote_up.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="periodComboBox">
           <item>
            <property name="text">
             <string>Today</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yesterday</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Week</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Month</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>This Year</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="GxsIdChooser" name="idChooser"/>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>711</width>
            <height>16</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GroupTreeWidget</class>
   <extends>QWidget</extends>
   <header>gui/common/GroupTreeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GxsIdChooser</class>
   <extends>QComboBox</extends>
   <header>gui/gxs/GxsIdChooser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Posted_images.qrc"/>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
